### DISCUSSION TITLE
default port solis dongle

### DISCUSSION THREAD

**Discussion by kiloptero** [Pho3niX90/solis_modbus#31] [Q&A]
hii;) thanks for this amazing hace, its possible to know the default port to put in the integration?

its 502?

**Comment by TheManoeuvre** [comment]
I also need this, I tried 502 to no avail

**Comment by 0rangutan** [comment]
502 is the correct default port, assuming that your Solis dongle is running on the standard port and that it hasn't already connected to Solis Cloud first.  The dongle can only handle one TCP connection and will go with your HA instance or Solis Cloud, whichever speaks to it first.  If you want to use both services at once, you need to set up something like this: https://github.com/alienatedsec/solis-ha-modbus-cloud
